src/models/Brand.ts

```diff
@@ -25,7 +25,6 @@ Brand.init(
         name: {
             type: DataTypes.STRING,
             allowNull: false,
-            unique: true,           // making the brand name unique
```

Comment:
why?

Reply:
Because when using Sequelize, there will be multiple references on the same column, which can cause the database to stop